app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < Mail::Notify::Mailer
+private
+
+  def blank_allowed(value)
+    value.presence || Mail::Notify::Personalisation::BLANK
+  end
```

Comment:
Can we open a PR in the gem that adds similar functionality? Or an issue?

Reply:
Done: https://github.com/dxw/mail-notify/pull/30